### PR TITLE
scripts/g.extension: fix get get_addons_paths func json_file attribute

### LINKED ISSUE #2093: [Bug] Failed to install r.futures from directory
**Describe the bug**
Installing r.futures (it's a multi-addon) from local directory fails:
```g.extension r.futures url=~/dev/grass/grass-addons/src/raster/r.futures/```

(from online addons repo it works)

```
/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension:167: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.dir_util import copy_tree
Compiling...
Installing...
Updating extensions metadata file...
Updating extension modules metadata file...
Traceback (most recent call last):
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension", line 2647, in <module>
    sys.exit(main())
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension", line 2625, in main
    install_extension(source=source, url=url, xmlurl=xmlurl, branch=branch)
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension", line 1043, in install_extension
    install_module_xml(new_modules)
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension", line 1345, in install_module_xml
    mlist = filter_multi_addon_addons(mlist)
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/scripts/g.extension", line 1300, in filter_multi_addon_addons
    get_addons_paths.json_file,
AttributeError: 'function' object has no attribute 'json_file'
```

**Additional context**
Seems to be related to #2090

### PR DESCRIPTION
**Pull request by tmszi**
Fixes #2093.

### CODE REVIEW

File: scripts/g.extension/g.extension.py [hunk 1]
```diff
@@ -2538,20 +2539,16 @@ def get_addons_paths(gg_addons_base_dir):
     in the $GRASS_ADDON_BASE dir. The file serves as a list of all addons,
```
Comment (petrasovaa) [inline scripts/g.extension/g.extension.py:2539]: Is there supposed to be addons_paths.json instead of extensions_paths.json?

**Review by petrasovaa — APPROVED** [review]
Seems to work, thank you!

File: scripts/g.extension/g.extension.py (hunk 1 above)
Comment (tmszi) [reply scripts/g.extension/g.extension.py]: Thanks for catching this. It looks like I'm confused by these two concepts extensions vs. addons.

Comment (veroandreo) [reply scripts/g.extension/g.extension.py]: > Thanks for catching this. It looks like I'm confused by these two concepts extensions vs. addons.

There was an open issue/discussion regarding the use of these terms. AFAIU, they are used indistinctly, i.e., addon=extension. We should probably choose one and stick to it for consistency. The module to install addons is called g.extension but the extensions repo is called addons 🤯🤪

**Pull request merged**